internal/testsuite/app.go

```diff
@@ -232,6 +233,21 @@ func (a *App) Test(ctx context.Context, testSpec *api.TestSpec, asserters ...fun
 	watcher.Out = a.Out
 	g.Go(func() error { return watcher.Run(ctx) })
 
+	// TODO: handle better
+	kubectxs := []string{
```

Comment:
I do not fully understand how and where will we run the testsuite, out-of-cluster or in-cluster, and how will we solve connectivity with Executor clusters